repos/fdbt-site/src/constants/attributes.ts

```diff
@@ -2,6 +2,10 @@
 
 export const FARE_TYPE_ATTRIBUTE = 'fdbt-fare-type';
 
+export const MATCHING_JSON_ATTRIBUTE = 'fdbt-matching-json';
```

Comment:
Could we move these to the bottom of the file? It makes them a little easier to find if we keep to the order (oldest at top and newest at bottom).

Reply:
Done